### ISSUE TITLE
please add tag before version with sstream

### ISSUE THREAD

**Issue by JojoS62** [Bwar/CJsonObject#28]
could you please add a release tag before the version that uses sstream? That version looks usable with microcontrollers, but adding sstream is pulling in too much stuff from c++ streams including exceptions, and that is increasing program size too much.

**Comment by Bwar** [comment]
ok, my negligence.

**Comment by JojoS62** [comment]
thanks for your fast replay.
And no, its ok, it depends on which systems your library is used.
I have compiled this now for a STM32F4 device with gcc and it uses about 30 kB in my debug build. The std::ostringstream is used only in the () operator. When this is not used, then the linker will not include the stream stuff. When I call the operator(), it adds 170 kB :) 
I have compiled the version 1.2 also, but there is a bracket mismatch in 
https://github.com/Bwar/CJsonObject/blob/28faaadacbf0cbda1c6eb295104b241e02c085af/CJsonObject.cpp#L2191
and 
https://github.com/Bwar/CJsonObject/blob/28faaadacbf0cbda1c6eb295104b241e02c085af/CJsonObject.cpp#L2253
